### PR TITLE
Add reshape to vertices

### PR DESCRIPTION
**Pull request by GeorgeNash**
Adds reshape to `Double`, `Integer` and `Bool` vertices.

Also adds an `(int...)` constructor to `ones` in `IntegerTensor` to keep it consistent.
Also adds a `trues` and `falses` constructor for `BooleanTensor`

### CODE REVIEW

File: keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/IntegerTensor.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ static IntegerTensor create(int[] values) {
         return create(values, 1, values.length);
     }
 
-    static IntegerTensor ones(int[] shape) {
+    static IntegerTensor ones(int... shape) {
```
Comment (icantsayrural) [inline keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/IntegerTensor.java]: 🎉

**Comment by icantsayrural** [comment]
Shouldn't the user just pass the vertex to a reshape vertex?

**Comment by GeorgeNash** [comment]
They can do, but people shouldn't really be creating new vertices. For example with multiplication we do `vertex.times(anotherVertex)` rather than `new MultiplicationVertex(vertex, anotherVertex)`. Both boil down to the same thing

**Comment by icantsayrural** [comment]
Ah, I didn't see we support that for other operation vertices. Makes sense.

File: keanu-project/src/test/java/io/improbable/keanu/vertices/bool/BoolVertexTest.java [hunk 2]
```diff
@@ -11,6 +11,9 @@
 
 import com.sun.org.apache.xpath.internal.operations.Bool;
 import io.improbable.keanu.tensor.bool.BooleanTensor;
+import io.improbable.keanu.tensor.intgr.IntegerTensor;
```
Comment (icantsayrural) [inline keanu-project/src/test/java/io/improbable/keanu/vertices/bool/BoolVertexTest.java]: Some unused imports

**Pull request merged**